### PR TITLE
Making plot tabs slightly more compact vertically

### PR DESCRIPTION
**Pull request by coderrsid**
Making plot tabs slightly more compact vertically so that it fits a 13 inch screen easily.
Made it compact, decrease font-size, decrease white spaces and moved the color samples options near the plot.

Changes proposed in this pull request:
- Made it compact
- Decreased font-size
- Removed extra white space
- Moved color sample options near plot.

# Screenshots
screenshot 2019-03-08 at 12 16 48 am
screenshot 2019-03-08 at 12 17 05 am

# Notify reviewers
@schultzn @jjgao

### CODE REVIEW

**Comment by jjgao** [comment]
@coderrsid thanks for the pull request. It's an interesting idea to have a floating menu. I think that might work well - we might want to tweak the position though, e.g align it to the bottom right of the plot.

It seems that the plot still does not fit, e.g. [this query](http://bit.ly/2VMPGwv). I think we should reduce the y-axis and even make it responsive (similar to CN Segments tab).

image

I noticed you changed the global `.tab-content`. If you look other tabs, the padding was necessary.

image

**Comment by jjgao** [comment]
any comments @adamabeshouse ?

**Comment by coderrsid** [comment]
ohh okay sure. I'll make the changes as you said. Then i'll make a new PR.

**Comment by schultzn** [comment]
The example fits on my screen. @jjgao are you sure you were at 100%?

**Comment by coderrsid** [comment]
Yeah @schultzn  i use a 13'' macbook air so that fits in my screen. But i'll work for making the color samples by in the plot.

**Comment by jjgao** [comment]
@schultzn I was using the chrome mock screen -- it's probably not work.

@coderrsid please ignore my comment of "the plot still does not fit" and work on the other comments. Thanks.

**Comment by coderrsid** [comment]
@jjgao  @schultzn  I made the changes as proposed by you. Please review and tell me if anything has to change more.

**Comment by jjgao** [comment]
@coderrsid thank you! I think your change to Plots tab looks good!

@alisman do you have any comments on the UI change? If it looks good to you we should have some code review.

@coderrsid if it's not too much trouble, do yo think you can create separate pull requests for other issues so that we can review and merge them separately?

**Comment by coderrsid** [comment]
@jigao Sure. i'll make another pull request for the issue #5320 but can you first approve this one?

File: src/pages/resultsView/plots/PlotsTab.tsx [hunk 1]
```diff
@@ -1112,8 +1112,8 @@ export default class PlotsTab extends React.Component<IPlotsTabProps,{}> {
         }
         return (
             <div>
-                <hr/>
-                <h4>Utilities</h4>
+                {/* <hr/>
+                <h4>Utilities</h4> */}
```
Comment (adamabeshouse) [inline src/pages/resultsView/plots/PlotsTab.tsx]: remove comment

**Pull request closed without merging**